src/control/controlbehavior.cpp

```diff
@@ -161,19 +171,19 @@ double ControlAudioTaperPotBehavior::valueToParameter(double dValue) {
     } else if (dValue < 1.0) {
         // db + linear overlay to reach
         // m_minDB = 0
-        // 0 dB = m_neutralParameter
+        // 0 dB = m_dNeutralParameter
         double overlay = m_offset * (1 - dValue);
         if (m_minDB) {
-            dParam = (ratio2db(dValue + overlay) - m_minDB) / m_minDB * m_neutralParameter * -1;
+            dParam = (ratio2db(dValue + overlay) - m_minDB) / m_minDB * m_dNeutralParameter * -1;
```

Comment:
>>   Do you use a refactoring tool? This will perform all renaming automatically.

> What's that? I have the search&replace tool in my beloved geany :)

For instance Eclipse has a feature that renames a single function and all it's usages. A normal  search&replace will rename all functions of all classes. Maybe geany is smarter?